content/efcore/getting-started/samples/loading-related-data/include-and-then-include-method-chain/Program.cs

```diff
@@ -13,13 +13,13 @@ public static void Main()
                     .ThenInclude(c => c.Reader)
                 .Include(b => b.Author)
                 .Single(b => b.Title.Contains("Orient Express"));
-
-            Console.WriteLine(String.Format("{0}", book.Title));
-            Console.WriteLine("Checkout Records:");
-            foreach (CheckoutRecord checkoutRecord in book.CheckoutRecords)
-            {
-                Console.WriteLine("{0} {1} - Due {2}", checkoutRecord.Reader.FirstName, checkoutRecord.Reader.LastName, checkoutRecord.DueDate.ToString("MMMM dd, yyyy"));
-            }
+		   
+			Console.WriteLine("Book: {0}\n Author: {1} {2}", book.Title, book.Author.FirstName, book.Author.LastName);
```

Comment:
Use multiple `Console.WriteLine` statements instead of the newlines. It will make the code more readable.